src/plugins/intel_gna/src/optimizer/gna_pass_manager.cpp

```diff
@@ -62,6 +62,37 @@ std::shared_ptr<IPassManager> BasePass::getPassManager() {
     return sharedMgr;
 }
 
+static auto DoNotSkip = [](CNNLayerPtr) {
+    return false;
+};
+
+/**
+ * @brief Perform addition of two blobs values
+ */
+static void SumBlobs(Blob::Ptr& src_blob, Blob::Ptr& dst_blob) {
+    if (!src_blob || !dst_blob || src_blob->size() != dst_blob->size()) {
+        return;
```

Comment:
Does or Doesn't matter if operation was conducted?

Reply:
This should not happen. Changed to asserts.